.github/checkstyle.yml

```
@@ -0,0 +1,13 @@
+
+on:
+  pull_request:
+    branches: ["master", "develop"]
+    paths:
+      - '**/*.java'
+
+jobs:
+  style-check:
+    uses: PSMRI/.github/.github/workflows/checkstyle.yml@main
+    with:
+      java-version: '17'
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)